src/common/types/stripe.ts

```
@@ -48,3 +48,47 @@ export const invoiceLinkGetResponseSchema = z.array(
+
+export const createInvoicePostResponseSchemaUnion = z.union([
+  createInvoicePostResponseSchema,     // success: 201
+  createInvoiceConflictResponseSchema, // info mismatch: 409
+]);
+
+export type PostCreateInvoiceResponseUnion = z.infer<
+  typeof createInvoicePostResponseSchemaUnion
+>;
+
+export const createInvoicePostRequestSchema = z.object({
+  invoiceId,          // reuse your meta’d primitive from file 2
```

ESLint

[error] Delete `·········`

(prettier/prettier)